registration/api/v2/serializers.py

```diff
@@ -22,16 +22,21 @@
 import logging
 from typing import Dict
 
+import tldextract
```

Comment:
Please add `tldextract` into `base.in`([file](https://github.com/open-craft/opencraft/blob/master/requirements/base.in)) since we are using it directly it is better to directly pin it.

Reply:
Oh, that's a fair point! Thanks for noticing 👌